ground/gcs/src/plugins/boards_dronin/boards_dronin.pro

```diff
@@ -18,7 +18,9 @@ HEADERS += \
     pikoblx.h \
     playuavosd.h \
     seppuku.h \
-    seppukuconfiguration.h
+    seppukuconfiguration.h \
+    sprf3e.h \
```

Comment:
What do you think about sprf3evo rather than sprf3e? I find it a lot clearer what board it's referring to ([there are a __lot__ of SPR boards all of a sudden?!](http://seriouslypro.com/)).